src/webui/abstractwebapplication.h

```diff
@@ -64,7 +64,9 @@ class AbstractWebApplication : public Http::ResponseBuilder, public Http::IReque
     void resetFailedAttempts();
     void increaseFailedAttempts();
 
-    void printFile(const QString &path);
+    virtual int maxAllowedFilesize() const;
```

Comment:
just curious, why make it `virtual`?

Reply:
Base class method returns some predefined default value, the subclasses can redefine it (even has it configurable).